<launch>
    <node pkg="tf" type="static_transform_publisher" name="mock_world" args="0.5 0.0 1.0 0 0.0 0.0 world ra_base_link 1" />
</launch>
